src/librustdoc/formats/item_type.rs

```diff
@@ -58,6 +58,7 @@ pub(crate) enum ItemType {
     TraitAlias = 25,
     // This number is reserved for use in JavaScript
     // Generic = 26,
+    Test = 27,
```

Comment:
I think it's fine to distinguish functions from test functions, but just in case:

cc @notriddle